packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.scss

```diff
@@ -1,41 +1,132 @@
 @import '../../common/common';
 
-$rowHeight: 36px;
-$rowVerticalPadding: 10px;
+$rowHeight: 42px;
+$compactRowHeight: 32px;
+$rowVerticalPadding: 8px;
+$compactRowVerticalPadding: 6px;
+$isPaddedMargin: 24px;
+
+
+/* DetailsList Colors */
+// Default
+$detailsList-item-default-header-color: $ms-color-neutralPrimary;
+$detailsList-item-default-meta-color: $ms-color-neutralSecondaryAlt;
+$detailsList-item-default-background-color: $ms-color-white;
+
+// Hover
+$detailsList-item-hover-color: $ms-color-neutralPrimary;
+$detailsList-item-hover-meta-color: $ms-color-neutralSecondary;
+$detailsList-item-hover-color-background: $ms-color-neutralLighter;
+
+// Selected
+$detailsList-item-selected-color: $ms-color-neutralDark;
+$detailsList-item-selected-meta-color: $ms-color-neutralPrimary;
+$detailsList-item-selected-background-color: $ms-color-neutralLight;
+
+// Selected Hover
+$detailsList-item-selected-hover-color: $ms-color-black;
+$detailsList-item-selected-hover-meta-color: $ms-color-neutralDark;
+$detailsList-item-selected-hover-background-color: $ms-color-neutralQuaternaryAlt;
+
+// Focus
+$detailsList-item-focus-header-color: $ms-color-black;
+$detailsList-item-focus-background-color: $ms-color-neutralQuaternaryAlt;
+$detailsList-item-focus-meta-color: $ms-color-neutralDark;
+
+
+// Compact mode rules
+:global(.ms-DetailsList--Compact) {
+  .root {
+    min-height: $compactRowHeight;
+    line-height: $compactRowHeight - ($compactRowVerticalPadding * 2);
+    border: 0;
+  }
+
+  .cell {
+    min-height: $compactRowHeight;
+    padding: $compactRowVerticalPadding 0;
+  }
 
-// component slots
-$detailsList-item-hover-color: $listItemHoverColor;
-$detailsList-item-selected-color: $listItemSelectedColor;
-$detailsList-item-selected-hover-color: $listItemSelectedHoverColor;
+  .check {
+    padding: 8px;
+  }
+}
 
+// Default state
 .root {
-  @include focus-border();
+  @include focus-border(0, $ms-color-neutralSecondary);
 
+  border-top: 1px solid $ms-color-neutralLighter;
+  border-bottom: 1px solid $ms-color-neutralLighter;
+  background: $detailsList-item-default-background-color;
+  color: $detailsList-item-default-meta-color;
   display: inline-block;
   min-width: 100%;
   min-height: $rowHeight;
-  margin: 1px 0;
   vertical-align: top;
   white-space: nowrap;
   padding: 0;
   box-sizing: border-box;
 
-  background: none;
-  border: none;
   @include text-align(left);
   vertical-align: top;
   line-height: $rowHeight - ($rowVerticalPadding * 2);
+  margin-top: -1px;
```

Comment:
Not sure. Outline is set to transparent from the focus-border mixin; would you recommend to overwrite it, or leave it as is with border instead?